cloudformation/extraction.yaml

```diff
@@ -51,6 +55,35 @@ ExtractQueueDeadLetterQueue:
   Properties:
     KmsMasterKeyId: !GetAtt KmsKey.Arn
 
+TextractRole:
+  Type: AWS::IAM::Role
+  Properties:
+    AssumeRolePolicyDocument:
+      Version: "2012-10-17"
+      Statement:
+        - Effect: "Allow"
+          Principal:
+            Service:
+              - "textract.amazonaws.com"
+              - "lambda.amazonaws.com"
+          Action:
+            - "sts:AssumeRole"
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AWSLambdaExecute"
```

Comment:
This should not be needed, as this is NOT the role that the lambda uses, it's one Textract assumes to write to the topic.